### ISSUE TITLE
Incorrect library reference in "Shuffle and Split Data"

### ISSUE THREAD

**Issue by bvajresh** [udacity/DSND_Term1#19]
Code contains the following import statement
`from sklearn.cross_validation import train_test_split`

sklearn.cross_validation does not exist. Code should be
`from sklearn.model_selection import train_test_split`

**Comment by sshtomar** [comment]
Hi @bvajresh 
Thanks for pointing this out. You are right.

Previously (before v0.18), `train_test_split` was located in the `cross_validation` module:

`from sklearn.cross_validation import train_test_split`
However, now it's in the model_selection module:

`from sklearn.model_selection import train_test_split`
so you'll need the newest version.

To upgrade to at least version 0.18, do:

`pip install -U scikit-learn`
(Or pip3, depending on your version of Python). If you've installed it in a different way, make sure you use another method to update, for example when using Anaconda.
